src/other/utils.js

```diff
@@ -1,6 +1,4 @@
-const angular = require('angular-fix');
-
-export default {formlyEval, getFieldId, reverseDeepMerge, findByNodeName};
```

Comment:
Your arguments make sense. But even if you're exporting variables, I would prefer something like this:

```javascript
// imports
import something from 'something-else';

// variable declaration
const someVar = getSomeVar();

// export stuff (rely on function declaration hoisting)
export {someVar, someFunc};



// function declarations
function someFunc() {
}

function getSomeVar() {
  return 'foo';
}
```

I like this because it reads like a newspaper. Important, high-level stuff on top. Details below.

Not sure what the community's thought is on this, but I use it for my projects and I really like the "newspaper" mentality.